js/src/config/graphql/queries.js

```diff
@@ -0,0 +1,11 @@
+export const PRODUCTS_QUERY = `
```

Comment:
queries should be under src folder, not in config

Reply:
what should the folder structure under source here?